charts/ocis/templates/proxy/route.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.route.enabled }}
+{{- include "ocis.basicServiceTemplates" (dict "scope" . "appName" "appNameProxy" "appNameSuffix" "") -}}
+apiVersion: route.openshift.io/v1
+kind: Route
+{{ include "ocis.metadata" . }}
+{{- if .Values.ingress.labels }}
+{{ toYaml .Values.ingress.labels | indent 4 }}
```

Comment:
should we allow to configure route labels instead of inherit the ingress ones?

Reply:
Only on request. I currently don't see the use case where they should differ.